docs/API.TraversingOutboundAndInboundLinksWithXgAndYg.md

```diff
@@ -274,15 +274,25 @@ Much of the syntax for the **yg** flag is the same as for the **xg** flag, so pl
 The main differences between the **xg** flag and the **yg** flag are:
 
 * With the **yg** flag, you can also use the `<` operator to indicate inbound links.
-* You cannot use the _ wildcard with **yg** (as this would create a prohibitively expensive query).
-* On inbound expansions (indicated by the `<` operator), you can use all comparison operators in field filters (instead of just `:` and `::` for outbound expansions).
+* You cannot use the '_' wildcard with **yg** (as this would create a prohibitively expensive query).
 
 For example:
 
     <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership,organizationCity.mdaas:Orlando&format=ttl&with-data&yg=<hasImmediateParent.mdaas<hasImmediateParent.mdaas
 
 This query retrieves all infotons that point to the matched infotons through their immediate parents field, and also all infotons that point to the level 1 inbound links through their immediate parents field. In other words, it retrieves the child companies of the child companies of the matched infotons. 
 
+<a name="hdrYgPipe"></a>
+### Implementing Multiple Expansion Paths with the Pipe Operator ###
+
+If you want to expand along several paths using a single query, you can add several expansion expressions separated by the pipe ('|') operator.
+
+For example, the following query searches for organizations whose name contains "Marriott", and also retrieves Quote and Instrument infotons that are issues by those organizations. Note the use of the '|' operator to expand along two different inbound links.
+
+    <cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott&yg=<isQuoteOf.mdaas|<isIssuedBy.mdaas
```

Comment:
This example is not a good usecase for pipes, as it can be wrtten using regular commas instead. Let's change to 2 path differentiating by direction. E.g: `<field.ns|>anotherField.ns`, or something with more meat in it.

Reply:
Gilad, please suggest authentic code examples for these two cases. I have a hard time coming up with them myself (as you can see).

Reply:
/permid.org might not be the best data to demonstrate such features, as it is highly denormalized.
How bout an example like:

Person --placeOfBirth--> Location
Person <--hasTenant-- Residence --locatedIn--> Location

where we try to find all the location a person been at.
the yg query could be something like:

yg=<hasTenant>locatedIn|>placeOfBirth

WDYT?

Reply:
Not sure I understand this completely. Does it look for all locations a person has been at, or all people who have lived at a certain location? Isn't the initial query on Residence infotons (not Person infotons)? Can you please send me the entire query?

Also, can you send me a good example for gqp as well (your comment below)?

Reply:
Usage is the same. when you want to start on expansion with '>', and one with '<'.
regarding the suggested example, let's use some geonames data for locations:
http://sws.geonames.org/4140704/about.rdf
http://sws.geonames.org/5122525/about.rdf
and add some properties of our own, which results in [this](https://gist.github.com/hochgi/05af93055c555ae68f96189e957043b2)

Now, when the above is ingested to CM-Well, you could do a yg query as follows:
```
<CM-Well HOST>/example.org/Individuals/Donald_Trump?yg=<hasTenant.xmpl>location.locn|>birthPlace.madsrdf
```

Does this make sense?